Shared/Reusable/ListButton.swift

```diff
@@ -24,6 +27,6 @@ struct ListButton: View {
                 #endif
             }
         }
-        .foregroundColor(.white)
+        .foregroundColor(colorScheme == .light ? .black : .white)
```

Comment:
This could be done with no need for the color scheme:
```suggestion
        .foregroundColor(.primary)
```

Reply:
Yeah, agreed here